packages/genesys-spark-tokens/reports/unused-defined-ui-tokens.json

```diff
@@ -614,6 +614,9 @@
   "--gse-ui-contextMenu-button-compact",
   "--gse-ui-contextMenu-button-default",
   "--gse-ui-copyToClipboard-gap",
+  "--gse-ui-dataTable-border-color",
+  "--gse-ui-dataTable-border-style",
+  "--gse-ui-dataTable-border-width",
```

Comment:
UX have a ticket to remove these deprecated tokens when they have finalised their deprecation process within their team.